src/discordas.ts

```diff
@@ -209,3 +224,70 @@ async function run(port: number, fileConfig: DiscordBridgeConfig) {
         process.exit(1);
     }
 }
+
+/**
+ * Logs an error which occured during event processing.
+ *
+ * Depending on the error type different log levels are hardcoded.
+ */
+function logOnEventError(err: Error): void {
```

Comment:
wouldn't it be cleaner to make this a private method? Same goes for the other defined methods below

Reply:
If I get TS right, this function is private to the module (file) and is not in the global namespace, so everything should be fine with it.